Library/Tracking/KSRAnalytics.swift

```diff
@@ -930,6 +932,40 @@ public final class KSRAnalytics {
     )
   }
 
+  /* Call when the Confirm button on the Risk Messaging modal is clicked
+
+   parameters:
+   - project: the project being pledged to
+   - reward: the chosen reward
+   - typeContext: The context of the pledge submit button for a project.
+   - checkoutData: all the checkout data associated with the pledge
+   - refTag: the associated RefTag for the pledge
+
+   */
+
+  public func trackPledgeConfirmButtonClicked(
+    project: Project,
+    reward: Reward,
+    typeContext: TypeContext,
+    checkoutData: CheckoutPropertiesData,
+    refTag: RefTag?
+  ) {
+    let props = projectProperties(from: project, loggedInUser: self.loggedInUser)
+      .withAllValuesFrom(checkoutProperties(from: checkoutData, and: reward))
+      // the context is always "newPledge" for this event
```

Comment:
but the context passed in is `pledgeConfirm`

Reply:
Good catch. Updated.